RN v12 Adjustments for Prod release (#900)
mia-platform/documentation: docs/release-notes/v12.0.0.md

image: "img/release-note-link-preview.png"
---

_November 30th, 2023_

**Welcome to Mia-Platform v12!**

:::info

> Mia-Platform Console v12.0.0 is **now in Generally Available!**

Author: ```suggestion
Mia-Platform Console v12.0.0 is **now Generally Available!**
```

To learn more about, check out the dedicated [documentation page](/runtime_suite/files-service/configuration.mdx).

#### Form Service BE & FE - v2.0.0

The Form Service now supports form start date, custom expiration message for each form and custom recap email sender.
The `FormDatePicker` now prevents past date selection and the form modal messages (concerning availability and expiration) are translatable.
Moreover, the `isEditable` default value has been fixed, along with an issue in the builder call that retrieves templates.

##### Breaking Changes

Please note that this update introduces a breaking change, since now it is mandatory to configure specific properties 
in the `form_schemas` collection (for further details, please refer to the Form Service documentation).

#### Messaging Service - v1.6.0

The `POST /send` endpoint now supports the new field `emailSender` to overwrite the sender set in the service config.

## How to update your Console

> For self-hosted installations, please head to the [self hosted upgrade guide](/self_hosted/installation-chart/100_how_to_upgrade.md#v12---version-upgrades) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `vX.X.X`.

Author: ```suggestion
For self-hosted installations, please head to the [self hosted upgrade guide](/self_hosted/installation-chart/100_how_to_upgrade.md#v12---version-upgrades) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `v11.0.3`.
```

## How to update your Console

> For self-hosted installations, please head to the [self hosted upgrade guide](/self_hosted/installation-chart/100_how_to_upgrade.md#v12---version-upgrades) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `v11.X.X`.

Author: ```suggestion
For self-hosted installations, please head to the [self hosted upgrade guide](/self_hosted/installation-chart/100_how_to_upgrade.md#v12---version-upgrades) or contact your Mia-Platform referent and upgrade to _Console Helm Chart_ `vX.X.X`.
```
